src/engine/identification.md

```diff
@@ -0,0 +1,76 @@
+# Engine API -- Client Version Specification
+
+Engine API structures and methods specified for client version specification.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Client Version Specification](#client-version-specification)
+- [Structures](#structures)
+  - [ClientCode](#clientcode)
+  - [ClientVersionV1](#clientversionv1)
+- [Methods](#methods)
+  - [engine_getClientVersionV1](#engine_getclientversionv1)
+    - [Request](#request)
+    - [Response](#response)
+    - [Specification](#specification)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Client Version Specification
+
+To facilitate a more accurate measurement of execution layer client diversity statistics, execution clients **SHOULD** support the methods described in this document.
+
+## Structures
+
+### ClientCode
+
+This enum defines a standard for specifying a client with just two letters. Clients teams which have a code reserved in this list **MUST** use this code when identifying themselves. The code is specified here only to facilitate standardization and NOT to imply that these are the only supported Ethereum clients. Any clients not listed here are free to use any two letters which don't collide with an existing client code. They are encouraged to make a PR to this repo to reserve their own code. Existing codes are as follows:
+
+ - `BU`: besu
+ - `EJ`: ethereumJS
+ - `EG`: erigon
+ - `GE`: go-ethereum
+ - `GR`: grandine
+ - `LH`: lighthouse
+ - `LS`: lodestar
+ - `NM`: nethermind
+ - `NB`: nimbus
+ - `TK`: teku
+ - `PM`: prysm
+ - `RH`: reth
+
+### ClientVersionV1
+
+This structure contains information which identifies a client implementation. The fields are encoded as follows:
+
+- `code`: `ClientCode`, e.g. `NB` or `BU`
+- `clientName`: `string`, Human-readable name of the client, e.g. `Lighthouse` or `go-ethereum`
```

Comment:
```suggestion
- `name`: `string`, Human-readable name of the client, e.g. `Lighthouse` or `go-ethereum`
```
Slight pref for "name". It's implied it is the client. There isn't "clientCode", "clientVersion", etc.